docs/source/02_installation.rst

```diff
@@ -168,6 +168,23 @@ If a hook identifies an issue (signified by the pre-commit script exiting with a
 
 Further details about pre-commit hooks can be found in the Contributors Guidelines, specifically in the provided in the :ref:`Code quality control <code_quality_control>` section.
 
+Download the test data
+......................
+
+Some of GPM-API's tests require additional data to be downloaded.
+Ensure you have ssh keys set up for your GitHub account.
+If you did not yet clone GPM-API on your computer, run:
```

Comment:
I would avoid to specify this. Contributors must fork their repository, not the main one. 
I would suggest:
After having clone your forked repository, go in to the gpm_api directory and run git submodule update --init --recursive